test/rockpaperscissors.js

```diff
@@ -0,0 +1,913 @@
+            await timeMachine.advanceTimeAndBlock(DAY_24_HOUR_IN_SECS);
+
+            const txObj = await rcp.playerTwoCollectsForfeit(gameKey, {from: playerTwo});
+
+            await truffleAssert.eventEmitted(txObj, 'ForfeitPaid', (ev) => {
+
+                return  ev.player === playerTwo &&
+                    ev.gameKey === gameKey &&
+                    ev.amount.toString(10) === "20"
+            },'ForfeitPaid event is emitted');
+
+            const playerOneOwed = await rcp.balances(playerOne);
+            const playerTwoOwed = await rcp.balances(playerTwo);
+
+            assert.strictEqual(playerOneOwed.toString(10), "0");
+            assert.strictEqual(playerTwoOwed.toString(10), "20");
+
+        });
+
+        it("Player two does not reveal and player one collects forfeit", async () => {
+
+            const gameKey = await rcp.createPlayerOneMoveHash(playerOne, playerOneSecretBytes32, SCISSORS);
+            await rcp.movePlayerOne(gameKey, "10", {from: playerOne, value: 10});
+            const playerTwoMoveHash = await rcp.createPlayerTwoMoveHash(playerTwo, gameKey, playerTwoSecretBytes32, ROCK);
+            await rcp.movePlayerTwo(gameKey, playerTwoMoveHash, {from: playerTwo, value: 10});
+            await rcp.revealPlayerOne(playerOneSecretBytes32, SCISSORS, {from: playerOne});
+            await timeMachine.advanceTimeAndBlock(DAY_24_HOUR_IN_SECS);
+
+            const txObj = await rcp.playerOneCollectsForfeit(gameKey, {from: playerOne});
+
+            await truffleAssert.eventEmitted(txObj, 'ForfeitPaid', (ev) => {
+
+                return  ev.player === playerOne &&
+                        ev.gameKey === gameKey &&
+                        ev.amount.toString(10) === "20"
+            },'ForfeitPaid event is emitted');
+
+            const playerOneOwed = await rcp.balances(playerOne);
+            const playerTwoOwed = await rcp.balances(playerTwo);
+
+            assert.strictEqual(playerOneOwed.toString(10), "20");
+            assert.strictEqual(playerTwoOwed.toString(10), "0");
+
+        }).timeout(TIMEOUT);
+    });
+
+    it("Player two can stake their winnings in a subsequent game", async () => {
+
+        const gameKey = await rcp.createPlayerOneMoveHash(playerOne, playerOneSecretBytes32, ROCK);
+        await rcp.movePlayerOne(gameKey, "10", {from: playerOne, value: 10});
+        const playerTwoMoveHash = await rcp.createPlayerTwoMoveHash(playerTwo, gameKey, playerTwoSecretBytes32, PAPER);
+        await rcp.movePlayerTwo(gameKey, playerTwoMoveHash, {from: playerTwo, value: 10});
+        await rcp.revealPlayerOne(playerOneSecretBytes32, ROCK, {from: playerOne});
+        await rcp.revealPlayerTwo(gameKey, playerTwoSecretBytes32, PAPER, {from: playerTwo});
+
+        const initPlayerTwoOwed = await rcp.balances(playerTwo);
+        assert.strictEqual(initPlayerTwoOwed.toString(10), "20");
+
+        const playerMoveHash = await rcp.createPlayerOneMoveHash(playerTwo, playerOneSecretBytes32, ROCK);
```

Comment:
`const gameKey2 =`?

Reply:
I called the first instance `gameKey1` for continuity.